package.json

```diff
@@ -10,7 +10,7 @@
     "test": "npm run test:node && npm run test:browser",
     "test:node": "aegir test --ts -t node",
     "test:browser": "aegir test -t browser",
-    "build": "aegir build",
+    "prepare": "aegir build",
```

Comment:
This needs to be reverted

Reply:
Without this you can't depend on a branch of this repo as otherwise it has no types after install.  `prepare` is run during release so it shouldn't break anything?

Reply:
We can keep it, but I would not remove the build script. Perhaps just add a new one?

Reply:
Can do, but then it'll build twice during release since npm runs `prepare` and aegir runs `build`